ninja: add job limit example (#10494)
tldr-pages/tldr: pages/common/ninja.md

`ninja`

> - Build in the current directory, with 4 parallel jobs at max:

Reviewer: ```suggestion
- Build in the current directory, executing 4 jobs at a time in parallel:
```

> `ninja -j{{4}}`

Reviewer 2: ```suggestion
`ninja -j {{4}}`
```
